core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1141,6 +1141,8 @@ public PhysicalOperation visitWindow(WindowNode node, LocalExecutionPlanContext
                     sortKeyChannelForEndComparison = Optional.of(source.getLayout().get(frame.getSortKeyCoercedForFrameEndComparison().get()));
                 }
                 if (node.getOrderingScheme().isPresent()) {
+                    // the following fields are only used for frame type RANGE
+                    // in such case, there is a single sort channel
                     sortKeyChannel = Optional.of(sortChannels.get(0));
```

Comment:
In this case can we use `getOnlyElement` ? `getOnlyElement(sortChannels)`

Reply:
No, we can't. See my comment: https://github.com/trinodb/trino/pull/5639#discussion_r1846210241